docs/source/ecosystem/flash.rst

```diff
@@ -0,0 +1,76 @@
+Lightning Flash
+===============
+
+`Lightning Flash <https://lightning-flash.readthedocs.io/en/latest/>`_ is a high-level deep learning framework for fast prototyping, baselining, finetuning and solving deep learning problems.
+Flash makes complex AI recipes for over 15 tasks across 7 data domains accessible to all.
+It is built for beginners with a simple API that requires very little deep learning background, and for data scientists, Kagglers, applied ML practitioners and deep learning researchers that
+want a quick way to get a deep learning baseline with advanced features PyTorch Lightning offers.
+
+.. code-block:: bash
+
+    pip install lightning-flash
+
+-----------------
+
+Using Lightning-Flash in 3 Steps!
+---------------------------------
+
+1. Load your data
+-----------------
```

Comment:
I'd suggest following this primer.
https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#headings

for better side-bar

Reply:
Updated!